setup.py

```diff
@@ -28,12 +28,13 @@
         "pre-commit",
     ],
     "docs": [
-        "sphinx==4.3.1",
-        "sphinx_rtd_theme==1.0.0",
-        "sphinx_gallery==0.10.1",
-        "m2r2==0.3.1",
-        "mistune<2.0.0",
-        "Pillow==8.4.0",
+        "sphinx==5.1.1",
+        "sphinx_autodoc_typehints==1.19.2",
+        "sphinx_rtd_theme==0.5.1",
```

Comment:
do you know why this needed to be downgraded?

Reply:
for my own curiosity, removing that one pin:
```
    m2r2 0.3.3 depends on docutils>=0.19
    sphinx 5.1.1 depends on docutils<0.20 and >=0.14
    sphinx-rtd-theme 1.0.0 depends on docutils<0.18
```
so whatever, let's stick with your working set for now